File: Tugas2/python/main.py
from proc import proc
from ram import ram
from disk import disk
from pc import pc

# menggunakan construct
pc1 = proc("Intel",100)
pc2 = ram("4 GB", 200)
pc3 = disk("SSD","512 GB", 300)
data = pc(pc1,pc2,pc3)

#Output
data.printpc();

File: Tugas2/java/Main.java
public class Main{
	public static void main(String[] args){

		//Menggunakan Construct
		pc data = new pc();
        proc pc1 = new proc("Intel",100);
        Ram pc2 = new Ram("4 GB", 200);
        disk pc3 = new disk("SSD","512 GB", 300);
		
		//Menggunakan Set
		data.setprocpc(pc1);
	    data.setRampc(pc2);
	    data.setdiskpc(pc3);
		
		//Print Output
		data.printpc();
	}
	
}